src/org/jitsi/impl/neomedia/transform/fec/FECTransformEngine.java

```diff
@@ -118,16 +140,30 @@ public FECTransformEngine()
         // Assumption: all packets in pkts have the same SSRC
         Long ssrc = findSSRC(pkts);
         if (ssrc == null)
+        {
             return pkts;
+        }
 
-        FECReceiver fpt;
+        AbstractFECReceiver fpt;
         synchronized (fecReceivers)
         {
             fpt = fecReceivers.get(ssrc);
             if (fpt == null)
             {
-                fpt = new FECReceiver(ssrc, incomingPT);
-                fecReceivers.put(ssrc, fpt);
+                try
+                {
```

Comment:
I'd rather just have a flag which determines whether to use ulpfec or flexfec. When we get to a situation where there's an abundance of FEC implementations to choose from, then we can implement a generic solution :)

Reply:
done